ingest/mappings/dataset.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
Oddly, same thing with the mapping for person.

http://data.deepcarbon.net/es/dco/person/_mapping?pretty